src/ITR/data/vault_providers.py

```diff
@@ -1,40 +1,38 @@
-from . import ureg
```

Comment:
Probably should be:
```
from ..data import ureg
```
Definitely needs to be before `pint` and/or `pint_pandas`